Station/pycpt-seasonal-station-Chile.ipynb

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
@xchourio can we remove this notebook now?

Reply:
> @xchourio can we remove this notebook now?

I would say yes, that is, I think it's more convenient for DMC to use a single notebook.